docs/how-to-guides/integrating-autoware-with-your-vehicle.md

```diff
@@ -4,7 +4,7 @@
 
 Prerequisites for the vehicle:
 
-- An onboard computer that satisfies the [Autoware installation prerequisites](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites)
+- An onboard computer that satisfies the [Autoware installation prerequisites](../installation/autoware/source-installation/#prerequisites)
```

Comment:
You need to specify `.md` files.
https://github.com/autowarefoundation/autoware-documentation/actions/runs/3582086021/jobs/6025893812#step:3:477
![image](https://user-images.githubusercontent.com/31987104/204765458-58ad1267-4ac0-45b9-88e2-e199a483e356.png)


```suggestion
- An onboard computer that satisfies the [Autoware installation prerequisites](../installation/autoware/source-installation/index.md#prerequisites)
```